source/access/mfa_guide.rst

```diff
@@ -157,10 +163,13 @@ continuing:
 The indentation is not strictly necessary, but is recommended for readability.
 
 If you now connect to the cluster using your standard ``ssh`` command, the
-certificate will automatically be stored for as long as your agent lives. If
-you want to use apps that use ``ssh`` in the background (NX, FileZilla), you will
-have to do this connection to the cluster as well. You are free to log out of
-that session afterwards. 
+certificate will automatically be stored for as long as your agent lives. 
+
+If you want to use apps that use ``ssh`` in the background (NX, FileZilla), you 
+should also first inject an mfa certificate in your agent before trying to connect.
```

Comment:
Same comment as Maxime above, you could probably drop the 'mfa' in 'mfa certificate' here